skillful_network_client/src/app/home/news-feed/add-publication/add-publication.component.html

```diff
@@ -0,0 +1,17 @@
+
+<div class='card mat-elevation-z4' id="publication-card">
+    <mat-card>
+        <div class="div-principal">
+            <div>
+                <mat-card-title>
+                    <button mat-button (click)="openDialog()" id="title-publication"><span class="material-icons" id="create-icon">create</span> Créer un post</button>
+                </mat-card-title>
+            </div>
+            <div class="div-icones">
+                <div><span class="material-icons" id="post-icon">post_add</span></div>
```

Comment:
Could we rather use the `<i class="material-icons md-24" >post_add</i>` in order to inherit the style?